code/game/objects/items/devices/flash.dm

```diff
@@ -156,7 +156,7 @@
 
 
 /obj/item/flash/proc/revolution_conversion(mob/M, mob/user)
-	if(!ishuman(M) || !(user.mind in SSticker.mode.head_revolutionaries))
+	if(!ishuman(M) || !user?.mind.has_antag_datum(/datum/antagonist/rev/head))
```

Comment:
```suggestion
	if(!ishuman(M) || !user.mind?.has_antag_datum(/datum/antagonist/rev/head))
```

this will runtime on a mindless mob without this, think you may have put it in wrong place